src/Apps/Order/redirects.tsx

```diff
@@ -139,6 +153,15 @@ const redirects: RedirectRecord = {
         goToPaymentIfPaymentIsNotCompleted,
       ],
     },
+    {
+      path: "review/counter",
```

Comment:
Can we add some tests to our current `redirects` test file to capture this new case? 😄 

And also, thanks for adding it!! Nice to see this logic is easy to extend. 🎉